src/woff2_dec.cc

```diff
@@ -24,6 +24,7 @@
 #include <string>
 #include <vector>
 #include <map>
+#include <unordered_set>
```

Comment:
Err. I think that's because an early draft did and it managed to sneak through review. Good catch, will fix.